draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -344,10 +347,18 @@ catalog zone syntax (as they are treated as regular zones).
 However, for the purpose of catalog processing, consumers MUST ignore
 such broken catalog zones, and instead treat them as a regular DNS zone.
 
-Similary, when a catalog zone expires, it loses its catalog meaning and
+When a previously correct catalog zone becomes a broken catalog zone, because
+of an update through an incremental transfer or otherwise, it loses its catalog
+meaning and MUST be treated as a regular DNS zone.
+No special processing occurs, no previously by this catalog configured member
+are removed or reconfigured in any way.
+Processing of the catalog resumes again when the catalog turns into a correct
+catalog zone again, for example by an additional update (through IXFR or
+otherwise) fixing the catalog zone.
+
+Similarly, when a catalog zone expires, it loses its catalog meaning and
 MUST be treated as a regular DNS zone.
-No special processing occurs; in particular, member zones MUST NOT be
-deconfigured.
+No special processing occurs until the zone becomes fresh again.
```

Comment:
There was a concern that "regular DNS zone" sounds like the zone suddenly can be queried like a normal zone. How about:

```
... expires [or, further up: becomes a broken catalog zone], it is turned ineffectual. Catalog processing is ceased, and no special treatment occurs until the zone becomes fresh again.
```

Perhaps "ineffectual" is not the most understandable word, but my dictionary didn't yield a better one ... Anyway, just throwing in some inspiration.

Reply:
Thanks @peterthomassen , but you must have commented before I pushed commit 23a56a2 . I tried to address it by just removing text.